polling_stations/templates/base.html

```diff
@@ -60,62 +60,33 @@
     {% block page_content %}
     {% endblock page_content %}
 {% endblock content %}
+{% block mailing_list %}{% endblock mailing_list %}
```

Comment:
What does this line do?
Should it be an `include`?

Reply:
It removes the mailing list which is now added by default in dc_utils.